src/header.ts

```diff
@@ -0,0 +1,122 @@
+import {Flag, flagKeys, keys} from './util';
+import {FontWeight, TitleOrient} from './vega.schema';
+
+export const HEADER_TITLE_PROPERTIES_MAP = {
```

Comment:
Define type to be keyOf HeaderBase

Reply:
Where would `HeaderBase` be defined and what should go there?

Reply:
I mean `HeaderConfig`, sorry.

Reply:
Got it.